corehq/apps/domain/test_decorators.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.test.client import Client
+from corehq.util.test_utils import flag_enabled
+from corehq.apps.users.models import CouchUser
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.domain.decorators import _two_factor_required
+
+
+class TestDecorators(TestCase):
```

Comment:
This test isn't testing the decorators (so I'd maybe name it something different) and is only testing that your `if` statement is working correctly. It's good to have that test, but the test it replaces also made sure that the correct response code and template are being returned. It might be worth looking at the previous test and seeing how yours can also test those important parts of the code.